tests/installation/sles4sap_product_installation_mode.pm

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/perl -w
+use strict;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    assert_screen "sles4sap-product-installation-mode";
+    send_key "alt-p";    # Proceed with standard SLES installation
+    if (get_var("SLES4SAP_MODE") =~ /sles4sap/) {
+        send_key "alt-o";    # prOceed with standard SLES for SAP Application installation
+        if (get_var("SLES4SAP_MODE") eq 'sles4sap_wizard') {
+            send_key "alt-s";    # Start the SAP Installation Wizard right after the OS installation
+        }
+    }
+    save_screenshot;
+    send_key $cmd{"next"}, 1;
```

Comment:
please no 2nd argument to send_key. Rather have a wait_still_screen or screen_change - if you have to run this on a physical machine, this translates into sleep 19!